### PR TITLE
Tidy up keen

### PR DESCRIPTION
**Pull request by adambraimbridge**
cc @commuterjoy @tom-parker @wheresrhys

### CODE REVIEW

**Comment by adambraimbridge** [comment]
Note: When this is deployed to production, it shouldn't affect the current instance of keen.io. It'll save "clean" data to a new keen.io project: https://keen.io/project/565c783f96773d49f93fde8a

File: Makefile [hunk 1]
```diff
@@ -15,6 +15,8 @@ run: build-production
 	 export REDISGREEN_URL=`cat ~/.redis-spoor`; \
```
Comment (wheresrhys) [inline Makefile]: Add all these to config-vars app, then shouldn't need to rely on defining them in dotfiles and makefile

File: sinks/models/device.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const deviceModel = {
+	sourceContainer: "egest.annotations",
+	propertiesToImport: [
+		"deviceAtlas",
+		"geo",
+		"ua"
+	],
+	propertiesToTransform: [{
+		from: "user.layout",
```
Comment (wheresrhys) [inline sinks/models/device.js]: I would move these away from properties of user to properties of device

File: sinks/models/content.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+"use strict";
+const helpers = require('../../helpers');
+
+const contentModel = {
+	sourceContainer: "egest.annotations",
+	propertiesToTransform: [{
+		from: "content",
+		to: ""
```
Comment (wheresrhys) [inline sinks/models/content.js]: why is this being done?

File: sinks/models/content.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+	},
+	{
+		from: "countedCounted.isCountedContent",
+		to: "isCountedContent"
+	}]
+};
+
+module.exports = {
+	importData: (message) => helpers.importData(contentModel, message),
+	cleanData: (message) => helpers.cleanData(contentModel, message)
```
Comment (wheresrhys) [inline sinks/models/content.js]: maybe only decorate with v3 metadata for now. We don't really need v1 (or v2?)

File: sinks/models/meta.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+"use strict";
```
Comment (wheresrhys) [inline sinks/models/meta.js]: Maybe leave meta out for now as hopefully n-instrumentation will get merged in to o-tracking, so the naming may change. Also currently it's a bit of a dumping ground for a variety of unrelated stuff e.g. optout reason, domPath to a CTA

File: Makefile (hunk 1 above)
Comment (adambraimbridge) [inline Makefile]: https://github.com/Financial-Times/spoor-consumer/issues/37

File: sinks/models/content.js (hunk 4 above)
Comment (adambraimbridge) [inline sinks/models/content.js]: Added content.metadata.capi1 to the blacklist.
https://github.com/Financial-Times/spoor-consumer/pull/35/files#diff-bdf0998a3d9dc0693d27659b023ad4c1R14

File: sinks/models/meta.js (hunk 5 above)
Comment (adambraimbridge) [inline sinks/models/meta.js]: I've removed the meta model. Any meta data will be in the "annotations" property with other non-imported data.
